.travis.yml

```diff
@@ -31,3 +31,10 @@ install: beaver dlang install
 # can exhaust Travis VM memory. Because of this we override
 # this with -j1 to make sure only one DMD is ran at a time.
 script: beaver dlang make -rj1 all test
+
+deploy:
+    provider: script
+    script: beaver bintray upload -d sociomantic/tsunami/tools/neptune deb/*.deb
+    skip_cleanup: true
+    on:
+        tags: true
```

Comment:
This will run on every build.

Mmm, please do it like this instead:

https://github.com/sociomantic-tsunami/d1to2fix/blob/ab9df496b5dd644a1ab1a74f9f6531418ca38e90/.travis.yml#L52-L57

(take a look at the "artifacts" part)

Reply:
Ping :-)